### ISSUE TITLE
Label needed on "Activity filter" select on News Feed

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#379]
Area of site: News Feed
URL: https://hcommons-staging.org/activity/

**Description**
The "activity filter" select (`<select id="activity-filter-by">`) does not have a label. 

**Recommendation**

Either one of these will fix the issue:

1. Add a `title` attribute to the select
2. Add an `aria-label` attribute to the select
